praw/models/base.py

```diff
@@ -1,8 +1,9 @@
 """Provide the PRAWBase superclass."""
 from copy import deepcopy
-from typing import Any, Dict, Optional, TypeVar
+from typing import TYPE_CHECKING, Any, Dict, Optional
```

Comment:
This should be alphabetized:
```suggestion
from typing import Any, Dict, Optional, TYPE_CHECKING
```

Reply:
Isort sorted it this way. It seems that constants get imported before types.